code/modules/modular_computers/NTNet/NTNRC/conversation.dm

```diff
@@ -28,7 +28,7 @@ var/global/ntnrc_uid = 0
 			if(!C.silent && C.username != username && C.program_state == PROGRAM_STATE_BACKGROUND)
 				for (var/mob/O in hearers(2, get_turf(C.computer)))
```

Comment:
Because `hearers` only returns mobs, you could change these in cases where they cannot be replaced with `audible_message` to:
```suggestion
				for (var/H in hearers(2, get_turf(C.computer)))
					var/mob/O = H
```